dataland-frontend/src/components/resources/frameworkDataSearch/lksg/LksgDataModel.ts

```diff
@@ -1,5 +1,5 @@
 import { LksgData } from "@clients/backend";
-import { getDataset, DropdownDatasetIdentifier } from "@/utils/PremadeDropdownDatasets";
```

Comment:
Are the changes apart from the linting ones, due to changes in the developer tool?

Reply:
Ok. So do we need a PR in the developer repo to review those changes?

Reply:
Yes, I opened one now.

Reply:
ok, then open to review the changes there